<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vkUploadImagesForm</class>
 <widget class="QWidget" name="vkUploadImagesForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>326</width>
    <height>159</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QImageViewer - Vkontakte - Upload images</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Choose album:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QComboBox" name="albumUserBox"/>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton">
       <property name="minimumSize">
        <size>
         <width>27</width>
         <height>27</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="uploadButton">
       <property name="text">
        <string>Upload</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+Return</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="shortcut">
        <string>Esc</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="status">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>status</zorder>
  <zorder>progressBar</zorder>
  <zorder></zorder>
  <zorder>label_3</zorder>
  <zorder></zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
